_rules/SVG-image-has-name

```diff
@@ -0,0 +1,145 @@
+---
+name: SVG image has name
+description: |
+  This rule checks that each SVG image element that is included in the accessibility tree has an accessible name.
+
+success_criterion:
+- 1.1.1 # Non-Text Content
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any `SVG` element that is [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+### Expectation
+
+Each target element has a [non-empty](#non-empty) accessible name.
```

Comment:
I am wondering it will just be enough to say 
> Each target element has an accessible name.

Reply:
I have removed the non-empty reference. I haven't included the [letters or numbers][] definition because an accessible name already indicates that the (attribute value has content.